assets/javascript/app.js

```diff
@@ -76,6 +94,60 @@ $(document).ready(function () {
                 //Assign title with O*Net ID for use with other ajax calls
                 occupationTitleDiv.attr("value", occupationOnetCode);
 
+                const urlOnetCode = "https://api.careeronestop.org/v1/lmi/JSoak5q9cSjVtxE/" + occupationOnetCode + location
+        
+                $.ajax({
+                    url: urlOnetCode,
+                    method: "GET",
+                    headers: {
+                        // OUR API TOKEN
+                        Authorization: "Bearer EpjdrTPww1oYCMGKS8r1cJzQD/M+rH43tuZPAQfd6eJgZPa8XPe8G0N9zSEdD/lWCHT+A1wN+niAY+bSU18adA==",
+                        Accept: "application/json"
+                    }
+                }).then(response => {
+                    console.log(JSON.stringify(response));
+        
+                });
+
+
+
+                const urlProgramsbyOccpation = "https://api.careeronestop.org/v1/lmi/JSoak5q9cSjVtxE" + occupationOnetCode + location + "/50/0/0/0/0/0/0/0/0/5"
+
+                $.ajax({
+                    url: urlProgramsbyOccpation,
+                    method: "GET",
+                    headers: {
+                        // OUR API TOKEN
+                        Authorization: "Bearer EpjdrTPww1oYCMGKS8r1cJzQD/M+rH43tuZPAQfd6eJgZPa8XPe8G0N9zSEdD/lWCHT+A1wN+niAY+bSU18adA==",
+                        Accept: "application/json"
+                    }
+                }).then(response => {
+                    console.log(JSON.stringify(response));
+        
+                });
+                
+
+            } //End of for Loop
```

Comment:
Also, it looks like you ended the for loop for the search button on-click early (line 130). The for loop is supposed to end after lines 151-166 so the divs get added for each occupation.